src/faqs/benchmark.md

```diff
@@ -0,0 +1,21 @@
+# Benchmark
+
+`pgvecto.rs` has been focused on performance from the beginning. We have continuously monitored the performance of `pgvecto.rs` and compared it with other vector search libraries. The following benchmark results are from January 2024.
+
+The test is done on Google Cloud [n2-standard-8](https://cloud.google.com/compute/docs/general-purpose-machines#n2_series) (8 vCPUs, 32 GB RAM) with the `laion-768-5m-ip` dataset.
+
+## `pgvecto.rs` v.s. `pgvector`
+
+With the HNSW index, `pgvecto.rs` can achieve **2.5x** responses per second as `pgvector` can do with a slightly better precision of around 97%. This advantage increases when higher precision is required.
+
+![pgvecto.rs_vs_pgvector](./images/2024Jan_pgvectors_compare.png)
+
+With the filter enabled, `pgvecto.rs` can always reach more than **2x** responses per second as `pgvector` can do with different filter probabilities.
+
+![pgvecto.rs_vs_pgvector_filter](./images/2024Jan_pgvectors_filter_compare.png)
+
+## `pgvecto.rs` with different quantization methods
+
+`pgvecto.rs` supports several [quantization methods](../usage/quantization.md). These methods can help to reduce memory usage at different scales. The following figure shows the performance of `pgvecto.rs` with different quantization methods.
```

Comment:
Can you add the link to fp16 vector type?

Reply:
I'm confused why these types are not supported by SQ. cc @VoVAllen

Reply:
fp16 is a data type. User can store embedding in postgres use fp16 format with half storage, and index is also in fp16.

For scalar quantization, it uses int8 to store the vectors. However, there're several implementation problems.
- It store both original precision vectors and int8 vectors. The original vector is used to calculate the max/min on each dimension. So the memory and index storage used now is actually original vector + int8 vector (they are on mmap, so can be swapped to disk if insufficient memory).
- As need max/min on each dimension, it depends on the global distributions. I think it's unnecessary since most vector's dimensions are equivalent. Therefore I prefer the int8 https://github.com/tensorchord/pgvecto.rs/pull/326 over the current scalar quantization implementation.

Reply:
>  I think it's unnecessary since most vector's dimensions are equivalent.

Do you mean most of the distributions are equal?

* Have you tested the precision in different datasets?
* Does that mean users have to convert themselves?